src/shared/utils.ts

```diff
@@ -329,7 +329,12 @@ export function isVideo(url: string) {
 }
 
 export function validURL(str: string) {
-  return !!new URL(str);
+  try {
+    new URL(str);
+    return true;
+  } catch (_) {
```

Comment:
The discard parameter can be omitted entirely here

```jsx
try {
  new URL(str);
  return true;
} catch {
  return false;
}
```

Reply:
Good point! I'll make that change too